pipelines/vars/forklift_foreman.yml

```diff
@@ -11,7 +11,6 @@ server_box:
       bats_tests:
         - "fb-verify-packages.bats"
         - "fb-test-foreman.bats"
-        - "fb-test-puppet.bats"
```

Comment:
I guess we can just ignore that this is kinda relevant to current releases?

Reply:
No?

Just do:
```yaml
- "{{ 'fb-test-puppet.bats' if pipeline_version != 'nightly' and pipeline_version is version('3.1', '<') else '' }}"